e2e/testcases/stress_test.go

```diff
@@ -261,6 +265,91 @@ func TestStressLargeRequest(t *testing.T) {
 	}
 }
 
+func TestStress100CRDs(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.Reconciliation1, ntopts.Unstructured, ntopts.StressTest,
+		ntopts.WithReconcileTimeout(configsync.DefaultReconcileTimeout))
+
+	// Apply 100 small CRDs on the cluster to make sure Config Sync can still work.
+	for i := 1; i <= 100; i++ {
+		crd := fakeCRD("anvil", fmt.Sprintf("acme-%d.com", i))
+		if err := nt.KubeClient.Create(crd); err != nil {
+			nt.T.Fatal(err)
+		}
+	}
+
+	nt.T.Log("restarting the reconciler Pod to discovery the new 100 CRDs in the controller startup time")
```

Comment:
Are we concerned about stress testing the `reconciler-manager` pod as well, or just the `reconciler` pod?

Reply:
We only want to stress the reconciler Pod. Restarting the reconciler Pod here is because the discovery client pulls the fresh schemas from the openapi when creating the reconciler in the startup time.

We can also wait until it refreshes the schema at the validation point, but the OOMKills we observed is at the startup time, so I feel it's better to simulate the same scenario.

Reply:
Ack, SGTM. I have no issue with restarting the `reconciler` Pod, was mainly curious if we might want to restart the `reconciler-manager` Pod as well